packages/react/src/transactions/useUpdateFollowPolicy.ts

```diff
@@ -20,6 +20,47 @@ export type UpdateFollowPolicyArgs = {
  *
  * You MUST be authenticated via {@link useLogin} to use this hook.
  *
+ * To setup a {@link FollowPolicyType.CHARGE} you need to define an amount of a currency as a fee.
+ * You can get a list of supported currencies via {@link useCurrencies}.
+ *
+ * @example
+ * Anyone can follow.
+ * ```tsx
+ * const { execute, loading, error } = useUpdateFollowPolicy();
+ *
+ * await execute({
+ *   followPolicy: {
+ *     type: FollowPolicyType.ANYONE,
+ *   },
+ * });
+ * ```
+ *
+ * @example
+ * No one can follow.
+ * ```tsx
+ * const { execute, loading, error } = useUpdateFollowPolicy();
+ *
+ * await execute({
+ *   followPolicy: {
+ *     type: FollowPolicyType.NO_ONE,
+ *   },
+ * });
+ * ```
+ *
+ * @example
+ * Anyone can follow, but they must pay a fee.
```

Comment:
Can you please add a little explainer about `Amount` with link like in here: https://github.com/lens-protocol/lens-sdk/blob/lens-v2/packages/react/src/transactions/useCreatePost.ts#L170-L183

Copy and paste is fine. Better over communicate at this point.

And also clarify the example lile in `useCreatePost` (see link above).

Reply:
done in https://github.com/lens-protocol/lens-sdk/pull/610/commits/daf67ba715df13142221e2de80356d54edf4bb91